Lection2/task1.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import "fmt"
+
+func median(i []int) float64 {
+	// Defining one extra variable > calling array length 3 times
+	var arrLength = len(i) / 2
+
```

Comment:
This function returns a median from the input slice. The slice itself can be unsorted. Please add a sorting of the input slice.